pinecone/sparse/splade/splade_vector_generation.ipynb

```diff
@@ -0,0 +1,8784 @@
+{
```

Comment:
<p>"Orgnise"-&gt;"Organize|Organise"</p><p>"align to" -&gt; "align with the"</p><p>Add period to end of sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>